packages/devtools_app/test/vm_developer/vm_developer_test_utils.dart

```diff
@@ -81,11 +83,96 @@ const testPos = SourcePosition(line: 10, column: 4);
 
 final testInstances = InstanceSet(instances: null, totalCount: 3);
 
+final testRequestableSize = InstanceRef(
+  kind: '',
+  identityHashCode: null,
```

Comment:
You'll want to rebase and run `flutter pub upgrade`. You shouldn't need to explicitly pass `null` to these constructors anymore so you'll get an analysis error when the CI checks run.

Reply:
Thanks for letting me know!